### PR TITLE
Update 1049 Title to appropriate status per ECIP-1000

### PR DESCRIPTION
**Pull request by gitr0n1n**
+ Add WIP designation due to lack of ECIP-1000 compliance.
+ Add ECIP Editor Notes for sections not in compliance to ECIP-1000
+ Add reference link to CDC 22 Article.

### CODE REVIEW

**Comment by meowsbits** [comment]
LGTM, pending approval by either of the authors.

**Comment by gitr0n1n** [comment]
> LGTM, pending approval by either of the authors.

Great, here is the author's quote:
"As a Draft proposal I would be happy to agree that it is still a work-in-progress." - @bobsummerwill 

https://github.com/ethereumclassic/ECIPs/issues/460#issuecomment-1024611364

**Review by IstoraMandiri — APPROVED** [review]

**Pull request merged**
